internal/pkg/aws/cloudwatch/cloudwatch_test.go

```diff
@@ -74,12 +75,45 @@ func TestCloudWatch_AlarmsWithTags(t *testing.T) {
 		"return if no alarms found": {
 			setupMocks: func(m cloudWatchMocks) {
 				m.rg.EXPECT().GetResourcesByTags(cloudwatchResourceType, gomock.Eq(testTags)).Return([]*rg.Resource{}, nil)
-				m.cw.EXPECT().DescribeAlarms(gomock.Any()).Return(nil, nil)
 			},
 
 			wantAlarmStatus: nil,
 		},
-	}
+		"success": {
```

Comment:
1. should we add a test case for the scenario where there are no alarms found with `GetResourcesByTags`?
2. Can we make the test case string more verbose, for example:
```suggestion
		"should invoke DescribeAlarms on alarms that have matching tags": {
```

Reply:
I added a "return if no alarms found" test case. Did you mean something else/more?

Reply:
Oh wait, haha, I didn't add it. I just changed it so that if `GetResourcesByTags` doesn't return anything, `DescribeAlarms` isn't called.